### ISSUE TITLE
Game screen icon has a tan background

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/fractions-common#106]
**Test device:**
Dell
**Operating System:**
Win 10
**Browser:**
Chrome
**Problem description:**
For https://github.com/phetsims/QA/issues/368 and https://github.com/phetsims/QA/issues/367
I noticed that, even though the background for the games is white, the icons in the home page of Matcher and in the home page and nav bar of Equality are tan.  This looks particularly odd in Equality.  Very minor

**Screenshots:**
![alsotan](https://user-images.githubusercontent.com/41024075/61412509-4ceabb00-a8a6-11e9-8830-5c43e740ab17.png)
![tanbackground](https://user-images.githubusercontent.com/41024075/61412515-5116d880-a8a6-11e9-934b-ef387aea7cf8.png)

**Troubleshooting information (do not edit):**

<details>

Name: ‪Fractions: Equality‬
URL: https://phet-dev.colorado.edu/html/fractions-equality/1.1.0-rc.2/phet/fractions-equality_all_phet.html
Version: 1.1.0-rc.2 2019-07-15 16:24:56 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.142 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by amanda-phet** [comment]
@jonathanolson are these done in code (I can't remember)? Could the background be changed? 

In Fraction Matcher, the nav bar icons don't have a background at all, but we can't do that in Fractions: Equality. 

I'd be fine with making them all have a white background if that's an easy change.

**Comment by jonathanolson** [comment]
Implemented the tan => white change above (I had copied the colors from the original fraction-matcher version).

Can you check to see if this looks good in master? Additionally, let me know if the fraction-matcher icons should have a white background (I prefer the transparent background that they currently have).

Once this looks good, I can move the patch to the RC branch.

**Comment by KatieWoe** [comment]
Looks good in master to me.

**Comment by amanda-phet** [comment]
I agree, I like the transparent background we are using in Fraction Matcher.

The white looks fine to me. It's bright, but matches the screens.

**Comment by jonathanolson** [comment]
Moved patch to the release branches.
